README.md

```
@@ -611,38 +611,34 @@ modelconverter benchmark rvc4 --model-path <path_to_model.tar.xz>
 The command prints a table with the benchmark results to the console and
 optionally saves the results to a `.csv` file.
 
+> [!IMPORTANT]
+> **Device Connection Requirements for RVC4**: The device must be connected and accessible either using the [Android Debug Bridge (ADB)](https://developer.android.com/tools/adb) or via SSH for the benchmarking to work in the following cases:
+>
+> - When `--device-monitor` is enabled (requires ADB or SSH connection to the device to calculate power consumption)
+> - When benchmarking is conducted using the SNPE tools (with `--no-dai-benchmark`; default is `--dai-benchmark`)
+>
+> The tool can find the correct device automatically but you can also specify it with the `--device-id` flag.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)